docs/detections/rules-ui-create.asciidoc

```diff
@@ -681,11 +681,11 @@ Example using the mustache "current element" notation `{{.}}` to output all the
 [float]
 [[rule-response-action]]
 === Set up response actions (optional)
-Use Response Actions to set up additional functionality that will run whenever a rule executes.
+Use Response Actions to set up additional functionality that will run whenever a rule executes:
 
-The Osquery Response Action allows you to include live Osquery queries with a custom query rule. When an alert is generated, Osquery automatically collects data on the system related to the alert. Refer to <<osquery-response-action>> to learn more.
+* **Osquery**: Include live Osquery queries with a custom query rule. When an alert is generated, Osquery automatically collects data on the system related to the alert. Refer to <<osquery-response-action>> to learn more.
```

Comment:
I noticed that the link for <<osquery-response-action>> appears in italics in the preview but the on the following line 687 the <<automated-response-actions>> link is not italicized. Curious. Any idea why?

Reply:
@benironside That's honestly one of the biggest mysteries of AsciiDoc. I've noticed it several times and haven't ever figured out why it italicizes sometimes, and doesn't other times. It doesn't seem like something we can control, or at least not without a deep dive into our CSS stylesheet or other inner workings.